### ISSUE TITLE
No notifications when alarm triggered

### ISSUE THREAD

**Issue by maciej-or** [nielsfaber/alarmo#647]
### Alarmo Version

v1.9.7

### HA Version

2023.1.1

### Bug description

Alarmo doesn't send notification if opened sensor has no friendly name (KeyError: 'friendly_name')
regular name could be used in that case

### Relevant log output

```shell
Logger: homeassistant.util.logging
Source: util/logging.py:156
First occurred: 18 January 2023, 23:52:12 (9 occurrences)
Last logged: 16:37:14

Exception in async_alarm_state_changed when dispatching 'alarmo_state_updated': ('1673996146', 'armed_away', 'triggered') Traceback (most recent call last): File "/config/custom_components/alarmo/automations.py", line 121, in async_alarm_state_changed await self.async_execute_automation(automation_id, alarm_entity) File "/config/custom_components/alarmo/automations.py", line 187, in async_execute_automation parts.append(await self.async_get_open_sensor_string(entity_id, status, lang)) File "/config/custom_components/alarmo/automations.py", line 282, in async_get_open_sensor_string name = friendly_name_for_entity_id(entity_id, self.hass) File "/config/custom_components/alarmo/helpers.py", line 15, in friendly_name_for_entity_id if state and state.attributes["friendly_name"]: KeyError: 'friendly_name'
Exception in async_alarm_state_changed when dispatching 'alarmo_state_updated': ('1673996146', 'armed_home', 'triggered') Traceback (most recent call last): File "/config/custom_components/alarmo/automations.py", line 121, in async_alarm_state_changed await self.async_execute_automation(automation_id, alarm_entity) File "/config/custom_components/alarmo/automations.py", line 187, in async_execute_automation parts.append(await self.async_get_open_sensor_string(entity_id, status, lang)) File "/config/custom_components/alarmo/automations.py", line 282, in async_get_open_sensor_string name = friendly_name_for_entity_id(entity_id, self.hass) File "/config/custom_components/alarmo/helpers.py", line 15, in friendly_name_for_entity_id if state and state.attributes["friendly_name"]: KeyError: 'friendly_name'
```

**Comment by nielsfaber** [comment]
I'm sorry, I did not have any time to work on this project recently. I will investigate the issue.

**Comment by nielsfaber** [comment]
This is fixed in release v1.9.8.
